src/components/Navbar/index.css

```diff
@@ -1,40 +1,135 @@
 @import "../../styles/layout.css";
 @import "../../styles/breakpoints.css";
 @import "../../styles/colors.css";
+@import "../../styles/variables.css";
+
+$modal-height: 402px;
 
 .root {
-  display: flex;
-  flex-direction: row;
+  display: grid;
+  align-items: center;
   justify-content: space-between;
+  grid-template-columns: repeat(2, auto);
+  grid-template-rows: 1fr;
 
   padding-top: $spacing-28;
+
+  a {
+    text-decoration: none;
+  }
+
+  @include Breakpoint-mobileOnly {
+    padding-top: $spacing-16;
+  }
 }
 
-.brand {
-  text-decoration: none;
+.root * {
+  z-index: 2;
+}
+
+.root.isOpen {
+  grid-row-gap: $spacing-56;
+  grid-template-rows: repeat(2, auto);
+
+  @include Breakpoint-mobileOnly {
+    max-width: $content-width-mobile;
+
+    padding-right: calc($gutter / 2);
+    padding-left: calc($gutter / 2);
+    margin: 0 auto;
+  }
+}
+
+.brand,
+.brand svg {
+  height: 26px;
+
+  @include Breakpoint-mobileOnly {
+    height: 12px;
+  }
 }
 
 .links {
   display: flex;
   flex-direction: row;
 }
 
-.link {
+.menu-toggle {
```

Comment:
Font color has to change to `dark-blue` in company and services page.

Reply:
Noticed that today, forgot to change that